srcpkgs/mesa/template

```diff
@@ -147,21 +147,27 @@ else
 	configure_args+=" -Dgallium-vdpau=disabled -Dgallium-va=disabled -Dgallium-xvmc=disabled"
 fi
 
-# empty values introduced by leading comma are not allowed; the whole enumeration can be empty
-# _gallium_drivers is not affected by this problem
-_vulkan_drivers=${_vulkan_drivers/=,/=}
-_dri_drivers=${_dri_drivers/=,/=}
-
-configure_args+=" ${_gallium_drivers} ${_vulkan_drivers} ${_dri_drivers}"
-
 if [ "$_have_vulkan" ]; then
+	_have_zink=yes
 	configure_args+=" -Dvulkan-layers=device-select,overlay"
 	if [ "$_have_intel" ]; then
 		configure_args+=",intel-nullhw"
 	fi
 	subpackages+=" mesa-vulkan-overlay-layer"
 fi
 
+if [ "$_have_zink" ]; then
+	_gallium_drivers+=",zink"
+	makedepends+=" vulkan-loader"
+fi
+
+# empty values introduced by leading comma are not allowed; the whole enumeration can be empty
+# _gallium_drivers is not affected by this problem
+_vulkan_drivers=${_vulkan_drivers/=,/=}
+_dri_drivers=${_dri_drivers/=,/=}
+
+configure_args+=" ${_gallium_drivers} ${_vulkan_drivers} ${_dri_drivers}"
```

Comment:
Then here have only

```suggestion
configure_args+=" ${_gallium_drivers}"
```

instead

Reply:
Uh, it's visually better to keep drivers in same variable imo. So is it okay to keep configure_args same and move rest of the lines where it is?

Reply:
If you make a lot of changes, it makes it harder when looking at the commit to know what's related and what isn't (and more importantly, if a given block is changing or not). That's why I'd prefer to keep them to a minimum.

I was looking at your diff and confused why cleaning up `_vulkan_drivers` was moved, when it had nothing to do with the PR changes.